app.js

```diff
@@ -41,6 +41,28 @@ app.use(bodyParser.urlencoded({ extended: true }))
 var routes = require('./routes/main')
 app.use('/', routes)
 
+app.get('/bids', function(req, res, next){
```

Comment:
Take note @singjielim please put the routes in the correct routes file not in app.js